File: Practice-Contest-1/question1.java
import java.util.*;
public class question1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t>0){
            String n = sc.next();
            int k = sc.nextInt();
            int count = 0;
            
            char number[] = n.toCharArray();
            for(char num:number){
                if(num-'0' == k) count++;
            }
            System.out.println(count);
            t--;
        }
    }
}

File: Practice-Contest-1/question1.cpp
#include<iostream>
using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--) {
        string n;
        int k;
        cin >> n >> k;
        int count = 0;
        for(int i=0;i<n.length();i++){
            if(n[i]-'0' == k) count++;
        }
        cout<<count<<endl;
    }
    return 0;
}
